accounts/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+from .managers import MyUserManager
+
+
+class MyUser(AbstractBaseUser, PermissionsMixin):
```

Comment:
Can we also rename this model to like `Account`

Reply:
okay, I will work on this